.rubocop.yml

```diff
@@ -29,3 +29,6 @@ RSpec/DescribeClass:
 # Using 'let(:registration1)' in a unit test isn't necessarily unreasonable.
 RSpec/IndexedLet:
   Enabled: false
+
+Rails/LexicallyScopedActionFilter:
```

Comment:
Does this need to be disabled for all files? If it's just for a few files, can the before_action be refactored to comply with the cop?

Reply:
Unfortunately in trying to fix this I am now facing a conflict between two RuboCop rules:

Rails/LexicallyScopedActionFilter: This rule checks that actions are explicitly defined in the controller where a filter is applied. In my case, the new action is not explicitly defined in CopyCardsPaymentFormsController and the other two controllers, but I have a before_action filter that applies to it. This is why RuboCop is raising a warning.

Lint/UselessMethodDefinition: This rule checks for method definitions that don't do anything but call super with the same arguments that were passed to the method. If I define the new action in CopyCardsPaymentFormsController to satisfy the Rails/LexicallyScopedActionFilter rule, and the new action only calls super, then RuboCop will raise a warning for the Lint/UselessMethodDefinition rule.

I therefore have to disable one of these cops, unless you have another idea on how to resolve?

Reply:
Awkward. Maybe just disable for the specific method rather than disabling globally.